package.json

```diff
@@ -5,7 +5,7 @@
   "main": "main.js",
   "scripts": {
     "start": "electron .",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
What's this for?

Reply:
I found this out recently while I was doing unit testing. So jest does unit testing by creating a worker pool of child processes that run tests. From what I understand, that means several unit tests can be run at the same time. For some reason, this causes complications when doing setup and teardown in each test files. For example, in each test files, at the setup, I want to create the testing sqlite Database and seed the master tables (e.g MsBiome, MsDevelopment, etc.). At the teardown, I want to destroy the knex object (severing connection to the testing sqlite Database) and destroy the Database (deleting the .db file).

Problem is, when I tried running jest without the `--runInBand` option, it always causes error like cannot delete the .db file because it's still in use or it cannot seed the master tables because there is already data inside it because it breaks the unique Primary Key rule meaning the database is not successfully cleaned. This implies that unit tests in other file are ran before unit tests in one file are finished. Therefore, I used this option so that jest can run all tests serially in the current process.

For reference:
https://jestjs.io/docs/cli#--runinband